README.md

```diff
@@ -1,23 +1,56 @@
 # The Sixie Opinionated Django Cookiecutter
 
-This is a work in progress, but it is heavily inspired by the work
+This is a work in progress project for jumpstarting production-ready Django projects. It is heavily inspired by the work
 of [cookiecutter-django](https://github.com/pydanny/cookiecutter-django)
 and Daniel Roy Greenfeld. It also borrows heavily from the ideas in the
 PyPA [warehouse](https://github.com/pypa/warehouse) as I feel they have 
 made an amazing developer onboarding experience for a very complicated
 project.
 
-Creating a new project using this repo:
+Clone this repo:
 
     $ git clone git@github.com:sixfeetup/cookiecutter-sixiedjango.git
+
+Checkout to the working directory:
+
     $ cd cookiecutter-sixiedjango
-    ### checkout a branch if you need to
+
+Create a virtualenv:
+
     $ python3 -m venv .venv
+
+Activate the virtualenv you have just created:
+
     $ source .venv/bin/activate
-    $ pip install -r requirements.txt
+
+Install the requirements:
+
+    $ pip install-r requirements.txt
+
+Generate a new cookiecutter-django project:
+
     $ cookiecutter . -o [path to destination directory]
 
-Answer the questions, and you'll have your new project!
+Answer the prompts with your own desired options and you'll have your new project! For example:
```

Comment:
**issue** IMO we don't need to list all questions, as they will be added/removed and we will forget to update here. Just answer the questions should be fine.